runtime/PalletIds.md

```diff
@@ -0,0 +1,5 @@
+| Pallet   | Name              | ID       | Account ID                                       | Comment      |
+|----------|-------------------|----------|--------------------------------------------------|--------------|
+| treasury | Treasury          | py/trsry | 5EYCAe5ijiYfyeZ2JJCGq56LmPyNRAKzpG4QkoQkkQNB5e6Z | No more used |
+| treasury | LogionTreasury    | py/lgtrs | ?                                                |              |
+| treasury | CommunityTreasury | py/cmtrs | ?                                                |              |
```

Comment:
`py/` is for Parity, we could use another prefix. Will the account IDs for Logion and community treasuries be set later?

Reply:
Yes I will set those ID's later on.